sdk-lib/src/main/java/cash/z/ecc/android/sdk/internal/AndroidApiVersion.kt

```diff
@@ -4,7 +4,7 @@ import android.os.Build
 import androidx.annotation.ChecksSdkIntAtLeast
 import androidx.annotation.IntRange
 
-internal object AndroidApiVersion {
```

Comment:
What are you thinking?  I didn't want it to be considered a public API of the SDK which is why I stuck it under the internal package to start with.

Reply:
Yes, I thought that your intention was to open it to the public, when I saw the removed internal modifier. Now I see it was because of the AndroidApiVersion usage in the demo-app. Isn't it better to keep the internal modifier here and provide some maybe lightweight version of it in the demo-app too?